database/src/main/resources/db/migration/V1.65.0__NOTIF-619_webhook_sources_auths.sql

```diff
@@ -0,0 +1,51 @@
+-- Create the columns for the IDs of the authentications that will be created in Sources.
+
+ALTER TABLE
+    endpoint_webhooks
+ADD COLUMN
+    basic_authentication_id BIGINT
+;
+
+ALTER TABLE
+    endpoint_webhooks
+ADD COLUMN
+    secret_token_id BIGINT
```

Comment:
Could you merge these 2 in a single `ALTER` ?

```suggestion
ALTER TABLE
  endpoint_webhooks
ADD COLUMN
    basic_authentication_id BIGINT,
ADD COLUMN
    secret_token_id BIGINT
```

Reply:
Done, thanks!